queries.sql

```diff
@@ -252,3 +251,18 @@ WHERE vt.name = 'Maisy Smith'
 GROUP BY s.name
 ORDER BY COUNT(*) DESC
 LIMIT 1;
+
+/* project 5 */
+SELECT COUNT(*) FROM visits where animal_id = 4;
+
+SELECT * FROM visits where vet_id = 2;
+
+SELECT * FROM owners where email = 'owner_18327@mail.com';
+
+/* explain analyze on queries */
+ EXPLAIN ANALYZE SELECT COUNT (*) FROM visits WHERE animal_id=4;
+
+ EXPLAIN ANALYZE SELECT * FROM visits where vet_id = 2;
```

Comment:
👏 Great job with the initial analysis and the inclusion of screenshots. However, there's a specific step to follow as per the project requirements:

- [ ] Before creating the index, the execution time is 1564ms, which is more than 1s. Thats good. 👍 As stated in the [project requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_performance_audit.md#:~:text=The%20following%20queries%20are%20taking%20too%20much%20time%20(1%20sec%20%3D%201000ms%20can%20be%20considered%20as%20too%20much%20time%20for%20database%20query).), The query after using the index screenshot should show how much time the query takes AFTER the improvements introduced, and the execution time should be significantly lower than 1 second (1000 ms). Currently which is 1205ms. Please run the queries again and try your best to reduce the execution time.

- [ ] Please remember to run the query again and update your screenshots. 🙏

To meet this requirement, kindly follow the steps mentioned in the previous comment and in the project requirement. Then update your screenshot accordingly. This will ensure that your analysis aligns perfectly with the project specifications. Keep up the good work! 👍📊🕐

Reply:
Dear Code reviewer,
Good morning,

We send you here the Screenshots requested by the last reviewer.
Please kindly consider this review request since it is our last available review and we have tried our best in improving and optimizing  our execution time but sadly our laptop's process time are too fast.

**Screenshots**: <br/>

![queryBefore1](https://github.com/IvonneBenitesRodriguez/vet_clinic/assets/22691013/b21c61f3-a8af-4959-9d0c-7eec8f69f637)
![queryAfter1](https://github.com/IvonneBenitesRodriguez/vet_clinic/assets/22691013/6f780c8c-d4e3-4b91-805a-400cb4a4f2a2)
![ queryBefore2](https://github.com/IvonneBenitesRodriguez/vet_clinic/assets/22691013/22440336-bf21-4f41-a91b-073239463264)
![queryAfter2](https://github.com/IvonneBenitesRodriguez/vet_clinic/assets/22691013/7639f2c1-d667-4d79-9d10-0e7d2e4b81bd)


Thanks for your support,